src/tests/suite/SConscript

```diff
@@ -95,6 +95,8 @@ def scons():
     denv.Install('$PREFIX/bin/io_conf', Glob('io_conf/daos_io_conf_1'))
     denv.Install('$PREFIX/bin/io_conf', Glob('io_conf/daos_io_conf_2'))
     denv.Install('$PREFIX/bin/io_conf', Glob('io_conf/daos_io_conf_3'))
+    denv.Install('$PREFIX/bin/io_conf', Glob('io_conf/daos_io_conf_4'))
+    denv.Install('$PREFIX/bin/io_conf', Glob('io_conf/daos_io_conf_5'))
```

Comment:
probably also add these tests to CI test as well?

Reply:
Yes, may need some setting to backend for example need those tmp dirs (need to be different for different parallel CI tasks) . maybe need help from tester I am not sure how to set those.